Automattic/vip-go-mu-plugins
# Updating VIP dashboard dependencies to latest minor

## Description

This PR bumps VIP dashboard's Node dependencies to their latest minor ones. This dependency update also results in new generated compiled files.

## Changelog Description

### Plugin Updated: VIP Dashboard

Internal dependency updates.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Go to `wp-admin` > `VIP Dashboard`.
3. Everything should render normally.
